packages/gatsby-plugin-image/README.md

```diff
@@ -26,331 +32,225 @@ If you're using the new `GatsbyImage` in addition to `StaticImage`, you'll also
 module.exports = {
   plugins: [
     `gatsby-plugin-image`,
+    `gatsby-psource-filesystem`,
     `gatsby-plugin-sharp`,
-    // `gatsby-transformer-sharp`
+    // `gatsby-transformer-sharp` // Needed for dynamic images
   ],
 }
 ```
 
-# StaticImage
+## Using the Gatsby Image components
 
-This component is a new, simpler way to use Gatsby's image processing tools and components without needing to write GraphQL queries. It is designed for static images such as logos rather than ones loaded dynamically from a CMS.
+### Decide which component to use
 
-```js
-import React from "react"
-import { StaticImage } from "gatsby-plugin-image"
+The Gatsby Image plugin includes two image components: one for static and one for dynamic images. The simplest way to decide which you need to is to ask yourself: _"will this image be the same every time the component or template is used?"_. If it will always be the same, then use `StaticImage`. If it will change, whether through data coming from a CMS or different values passed to a component each time you use it, then it is a dynamic image and you should use the `GatsbyImage` component.
```

Comment:
```suggestion
The Gatsby Image plugin includes two image components: one for static and one for dynamic images. An effective way to decide which you need to is to ask yourself: _"will this image be the same every time the component or template is used?"_. If it will always be the same, then use `StaticImage`. If it will change, whether through data coming from a CMS or different values passed to a component each time you use it, then it is a dynamic image and you should use the `GatsbyImage` component.
```